### ISSUE TITLE
Add download links for different nanodegrees in Downloads folder

### ISSUE THREAD

**Issue by manishbisht** [manishbisht/Udacity#1]
Udacity offers various nanodegree. Create a file name inside downloads folder "Nanodegree Name.md" and write download links of the videos in it.

**Comment by sw-yx** [comment]
not sure if this is hacktoberfestworthy

**Comment by luisamiranda** [comment]
It's not much of a hack, but I'll take it.

**Comment by manishbisht** [comment]
Sure !! Submit the PR. :+1:

**Comment by NikoLewis** [comment]
Hey quick question do you only want these created for the nano degrees in the repo (front-end, full-stack, intro to programming) or all the possible nano-degrees on the Udacity site?

**Comment by manishbisht** [comment]
For all the nanodegrees from Udacity. You can submit one PR for each nanodegree.

**Comment by luisamiranda** [comment]
I created all the .md files, but then I can't get to the video links without enrolling in the course, can I?

**Comment by manishbisht** [comment]
No, You have to be enrolled to get the download links. Also you can take use of the 7 days trial to get the links.

**Comment by luisamiranda** [comment]
ok. i can do the free trial and finish this tomorrow.

**Comment by luisamiranda** [comment]
Sorry. I can't do the free trial without giving all my cc information, so I'm going to release this.

**Comment by Aashutosh7138** [comment]
Have you downloaded ML basic and advance all viodeos or these are only ML advance videos only
